client/app/kubernetes/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+    <translation id="kube_guides">Guides</translation>
+    <translation id="kube_guides_kubernetes">Kubernetes</translation>
+
+    <translation id="kube_error">Une erreur s'est produite lors du chargement des informations: {{ message }}</translation>
+
+    <translation id="kube_service">Service</translation>
+    <translation id="kube_service_description_information">Vous trouverez ci-dessous les informations de votre service Kubernetes, vous permettant d'y accéder à l'aide de l'</translation>
```

Comment:
I suspect that you're doing that in order split your sentence with an anchor inside. But as translator point of view that's not convenient to deal with such a thing.